### ISSUE TITLE
Click "Sign In" on checkout page block entire page

### ISSUE THREAD

**Issue by LordHansolo** [SnowdogApps/magento2-theme-blank-sass#106]
### Preconditions
1. Magento 2.1.x
2. Not logged in customer.
### Steps to reproduce
1. Add to Cart some products.
2. Go to checkout. Remember to be logged out.
3. Click on "Sign In" (sign in window is showing)
4. Click close on sign in window or on any place else than sign in window (sign in window is hiding)
5. Try click on any input or again "Sign In".
### Expected result

We can click again "Sign In" or fill any inputs on Page.
### Actual result

We can't click on any element visible on page.
### Additional informations

When we click "Sign In" there is showing aside sign in window with z-index=900 and  'div class="dropdown-overlay modal-custom-overlay" style="z-index: 899;"'. Our aside sign in window is with z-index:900 so we can close it but rest of the page is behind that overlay div. This overlay div should be removed when we close aside sign in window or click on that overlay div.

**Comment by aronesztergomi** [comment]
The reason is that the `.authentication-dropdown` has no CSS transition.

https://github.com/magento/magento2/blob/2.0/app/code/Magento/Ui/view/base/web/js/modal/modal.js#L280

So that event is never execute.

Hot fix:

```
.authentication-dropdown {
    transition-duration: 0.3s;
    -o-transition-duration: 0.3s;
    -moz-transition-duration: 0.3s;
    -webkit-transition-duration: 0.3s;
}
```
or

```
.authentication-dropdown {
   @include transition(0.3s);
}

**Comment by Igloczek** [comment]
How transition can fix this issue? It will not fix overlapping of elements.

**Comment by Skepton** [comment]
I think the logic was that the function listens for a transitionEvent to fire for that element, if the lement does not have a transition set, the event won't fire. But just a few more lines below in that same function the modal will be closed if there is no transitionEvent set. So that is most likely not the source of any issue.

**Comment by Leland** [comment]
Interestingly, I can confirm that fixing the transition properties does fix it.

With #152 merged, this should be closed.

**Comment by Igloczek** [comment]
Closing, b/c #152 fixes this issues.

Will be released with version 0.11.0
